src/helm/benchmark/augmentations/cleva_perturbation.py

```diff
@@ -715,7 +726,7 @@ class MandarinToCantonesePerturbation(Perturbation):
     should_perturb_references: bool = True
 
     """ Resources """
-    SOURCE_URI: str = "https://drive.google.com/uc?id=1vljbwq0hTm7W1tz74gjPnONWJ6kSEwK2"
+    SOURCE_URL: str = "http://39.108.215.175/assets/simplified_jyutping_conversion.json"
```

Comment:
You may need to rename this file to "conversion.json" in the file server to match the name in Line 734.